inc/modules/export/epub/class-epub201.php

```diff
@@ -734,6 +734,13 @@ protected function normalizeCssUrls( $css, $scss_dir, $path_to_epub_assets ) {
 					if ( $my_image ) {
 						copy( $my_image, "$path_to_epub_assets/$filename" );
 						return "url(assets/$filename)";
+					} else {
+						// This is not a Buckram theme, maybe.
+						$my_legacy_image = realpath( $scss_dir . '/' . $url );
```

Comment:
To be consistent with the code a few lines down, use` "$scss_dir/$url"` instead.

Also weird is sometimes we do `$my_image = realpath( "$scss_dir/$url" );` other times we do `$my_image = realpath( $buckram_dir . $url );`  (One has a slash, the other doesn't. I just want to flag it as something that bugs me. I am not saying refactor.

Reply:
It annoyed me too. I've made the method more consistent.